admin/etc/nginx.conf

```diff
@@ -1,24 +1,23 @@
 server {
-    include /etc/nginx/listen_ip.conf;
-    server_name onlineconf.example.com;
+    listen 80;
+    server_name oc.localhost;
 
-    access_log  /var/log/nginx/onlineconf.example.com-access.nginx.log main;
-    error_log   /var/log/nginx/onlineconf.example.com-error.nginx.log;
+#    access_log  /var/log/nginx/onlineconf.example.com-access.nginx.log main;
+#    error_log   /var/log/nginx/onlineconf.example.com-error.nginx.log;
 
     root /usr/local/www/onlineconf;
     charset UTF-8;
 
-    allow 192.0.2.0/24;
-    allow 198.51.100.0/24;
-    allow 203.0.113.0/24;
+    allow 172.0.0.0/8;
     deny all;
 
     location ~^/(?:debug|client|botapi|batch|config|whoami|user|group|access|search|global-log|log|monitoring|ui-config)(?:$|/) {
-        proxy_pass http://127.0.0.1:5000;
+        proxy_pass http://${NGINX_ADMIN_HOST}:80;
         proxy_set_header Host $http_host;
         proxy_set_header X-Forwarded-Host $host;
         proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+	proxy_set_header X-Forwarded-User ${NGINX_PROXY_USER};
```

Comment:
Why is it good for a new user to be able to log in only as one user without the ability to quickly log in as another in order to evaluate the system through the eyes of another role?
It can mislead a new user and make them think that onlineconf doesn't has nor authorization nor any RBAC.

Reply:
May be but this simply example.
Add basic auth to nginx. Is it looks good?